program-runtime/src/compute_budget.rs

```diff
@@ -118,9 +99,6 @@ pub struct ComputeBudget {
     pub alt_bn128_pairing_one_pair_cost_other: u64,
     /// Big integer modular exponentiation cost
     pub big_modular_exponentiation_cost: u64,
-    /// Maximum accounts data size, in bytes, that a transaction is allowed to load; The
-    /// value is capped by MAX_LOADED_ACCOUNTS_DATA_SIZE_BYTES to prevent overuse of memory.
-    pub loaded_accounts_data_size_limit: usize,
```

Comment:
tech debt - `loaded_accounts_data_size_limit` is not used by VM, it was added to `struct ComputeBudget` because instructions are processed here. Now it doesn't have to piggyback on `ComputeBudget` anymore.